department-of-veterans-affairs/caseflow
# Daily Docket - Saving to backend

Resolves #7076 

### Description
This PR saves edits to the daily docket to the backend. A future PR will handle the hearing date/time updates.

### Testing Plan
1. Go to `hearings/schedule/docket/${parent_record.id}`
1. Edit some fields
1. Confirm UI updates as expected
